### PR TITLE
CSP-21-Outer-ConfirmRegulatedStandard

### PR DESCRIPTION
**Pull request by raviudari01**

### CODE REVIEW

File: src/SFA.DAS.SharedOuterApi/Services/RoatpCourseManagementApiClient.cs [hunk 1]
```diff
@@ -28,5 +32,57 @@ public Task<ApiResponse<TResponse>> GetWithResponseCode<TResponse>(IGetApiReques
         {
             return _apiClient.GetWithResponseCode<TResponse>(request);
         }
+
+        public async Task<ApiResponse<TResponse>> PostWithResponseCode<TResponse>(IPostApiRequest request)
+        {
+            return await _apiClient.PostWithResponseCode<TResponse>(request);
+        }
+
```
Comment (MarkFCain) [inline src/SFA.DAS.SharedOuterApi/Services/RoatpCourseManagementApiClient.cs:40]: remove spaces

File: src/SFA.DAS.Roatp.CourseManagement.Api.UnitTests/Controllers/ProviderCourseEditControllerTests/UpdateProviderCourseContactDetailsTests.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            int ukprn, int larsCode, UpdateContactDetailsCommand command)
+        {
+            mediator.Setup(m => m.Send(It.Is<UpdateContactDetailsCommand>(c => c.Ukprn == ukprn && c.LarsCode == larsCode), It.IsAny<CancellationToken>())).ReturnsAsync(HttpStatusCode.NoContent);
+
+            var result = await sut.UpdateProviderCourseContactDetails(ukprn, larsCode, command);
+
+            (result as NoContentResult).Should().NotBeNull();
+        }
+
+        [Test, MoqAutoData]
```
Comment (MarkFCain) [inline src/SFA.DAS.Roatp.CourseManagement.Api.UnitTests/Controllers/ProviderCourseEditControllerTests/UpdateProviderCourseContactDetailsTests.cs:32]: Can you add the same test with a 'InternalServerError code too

Id suggest using TestCase() but you can also do it as a separate test.  Ping me if you want to discuss

File: src/SFA.DAS.Roatp.CourseManagement/InnerApi/Requests/UpdateContactDetailsRequest.cs [hunk 3]
```diff
@@ -0,0 +1,21 @@
+﻿using SFA.DAS.Roatp.CourseManagement.Application.Standards.Commands.UpdateContactDetails;
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.Roatp.CourseManagement.InnerApi.Requests
+{
+    public class UpdateContactDetailsRequest : IPostApiRequest
+    {
+        public int Ukprn { get; }
+        public int LarsCode { get; }
+        public string PostUrl => $"providers/{Ukprn}/courses/{LarsCode}/";
```
Comment (MarkFCain) [inline src/SFA.DAS.Roatp.CourseManagement/InnerApi/Requests/UpdateContactDetailsRequest.cs:10]: does this need to end in a '/'?  remove if not, and update test accordingly.

**Review by MarkFCain — APPROVED** [review]
minor comments to address, please mark as resolved as they are addressed

**Review by DevenKShah — DISMISSED** [review]

**Review by DevenKShah — APPROVED** [review]

**Pull request merged**